src/Cooked/Pretty/Cooked.hs

```diff
@@ -355,9 +355,12 @@ prettyTxSkelOut opts (Pays output) =
                   "Datum (inlined):"
                     <+> (PP.align . prettyCookedOpt opts)
                       (output ^. outputDatumL)
-              Api.OutputDatumHash _datum ->
+              Api.OutputDatumHash dHash ->
                 Just $
-                  "Datum (hashed):"
+                  "Datum (hashed)"
+                    <+> "("
+                    <> prettyHash (pcOptHashes opts) (toHash dHash)
+                    <> "):"
```

Comment:
Maybe we should have a flag in the prettyprinting options to enable this on demand and hide it by default. The default printing might be starting to get a bit crowded (this is not the reason, this is just a drop, it just makes me think about it). In general, we don't really track datum hashes and this would be lighter by default.

For instance `pcOptPrintDatumHashes` like there already is a `pcOptPrintTxHashes`, false by default.

Reply:
I will leave these changes to later changes in the pretty printer.